src/playerHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { willCollide } from './collidable';
+import { Enemy } from './enemy';
+import { knockback } from './movementHandler';
+import { PhysObject } from './physObject';
+import { Player } from './player';
+
+export function createPlayer(canvas: HTMLCanvasElement) {
+  const ctx = canvas.getContext('2d');
+  if (ctx === null) {
+    throw new Error('CanvasRenderingContext2D is null.');
+  }
+  return new Player(ctx, canvas.width / 2, canvas.height / 2, 20, 20, 100);
+}
+
+export function updatePlayer(player: Player, enemies: Array<Enemy>) {
```

Comment:
Conceptually, I feel like the logic currently in this function belongs in something like a collisionHandler file. It should also be more generic. Something like:

```
const allObjectsAfterMoving = getAllMovedObjects(); // need to figure this out, of course
const collidingObjectPairs = getCollidingObjects(allObjectsAfterMoving);
for (const {collidingObject, collidedObject} of collidingObjectPairs) {
  if (collidedObject.causesKnockback()) {
    handleKnockBack(collidingObject, collidedObject);
  }
}
```
}

Reply:
I've decided this would best be handled on it's own. Issue #3